zigzag/link_generation_facade.py

```diff
@@ -17,35 +18,8 @@ def __init__(self, mediator):
         Args:
             mediator (ZigZag): the mediator that stores shared data
         """
-        self._git_sha = None
+        self._git_sha = None  # TODO refactor this out
```

Comment:
Add tech debt ticket reference to make sure it gets done.

Reply:
ASC-1633